From 592af75987dcaf608e4f4a1b03b7910e3b5ae2be Mon Sep 17 00:00:00 2001
From: Haibo Chen <haibo.chen@freescale.com>
Date: Thu, 16 Apr 2015 18:32:22 +0800
Subject: [PATCH] MLK-10671 iio: ad2802a: use GPL v2 license

Correct the GPL license to v2, not V2. Otherwise
this driver is GPL-incompatible, and can't be make
as module.

Signed-off-by: Haibo Chen <haibo.chen@freescale.com>
---
 drivers/iio/adc/ad2802a.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/iio/adc/ad2802a.c b/drivers/iio/adc/ad2802a.c
index a8b08a0..5e565cb 100644
--- a/drivers/iio/adc/ad2802a.c
+++ b/drivers/iio/adc/ad2802a.c
@@ -561,4 +561,4 @@ module_platform_driver(ad2802_driver);
 
 MODULE_AUTHOR("Haibo Chen <b51421@freescale.com>");
 MODULE_DESCRIPTION("Freeacale ad2802 ADC driver");
-MODULE_LICENSE("GPL V2");
+MODULE_LICENSE("GPL v2");
-- 
1.8.0

